server/routerlicious/packages/kafka-orderer/src/kafkaOrderer.ts

```diff
@@ -131,6 +131,7 @@ export class KafkaOrdererConnection implements core.IOrdererConnection {
                 const operation = message.operation;
                 if (operation && operation.traces === undefined) {
                     operation.traces = [];
+                // eslint-disable-next-line @typescript-eslint/prefer-optional-chain
                 } else if (operation && operation.traces && operation.traces.length > 1) {
```

Comment:
```suggestion
                } else if (operation?.traces?.length > 1) {
```

I think this is equivalent.